test/PoolManager.t.sol

```diff
@@ -485,6 +489,117 @@ contract PoolManagerTest is TestSetup {
         centrifugeChain.updateTrancheTokenPrice(poolId, trancheId, currencyId, price, uint64(block.timestamp - 1));
     }
 
+    function testRemoveLiquidityPool()public {
+        address lPool_ = deploySimplePool();
+        LiquidityPool lPool =  LiquidityPool(lPool_);
+        uint64 poolId = lPool.poolId();
+        bytes16 trancheId = lPool.trancheId();
+        address currency = address(lPool.asset());
+
+        root.relyContract(address(poolManager), address(this));
+        poolManager.removeLiquidityPool(poolId, trancheId, currency);
+        assertEq(poolManager.getLiquidityPool(poolId, trancheId, currency), address(0));
```

Comment:
Would probably be good to add some checks here that after removing, the old LP is not a ward anymore on the investment mgr or tranche token, nor a trusted forwarder, and has 0 approval on the tranche token